gemcalibration/src/common/CalibrationWeb.cc

```diff
@@ -0,0 +1,310 @@
+    this->monitorPage(in, out);
+    *out << "    </div>" << std::endl;
+    *out << "    <div class=\"xdaq-tab\" title=\"Expert page\">" << std::endl;
+    this->expertPage(in, out);
+    *out << "    </div>" << std::endl;
+    *out << "</div>" << std::endl;
+
+    GEMWebApplication::webFooterGEM(in, out);
+
+    std::string updateLink = "/" + p_gemApp->m_urn + "/";
+
+    *out<<"<script type=\"text/javascript\">" <<std::endl
+        <<"store_actionURL(\""<<updateLink<<"\");"<<std::endl
+        <<"</script>"<<std::endl;
+    
+
+    
+}
+
+void gem::calib::CalibrationWeb::calibrationPage(xgi::Input* in, xgi::Output* out)
+{
+    CMSGEMOS_DEBUG("CalibrationWeb::calibrationPage");
+    *out << "<div class=\"xdaq-tab-wrapper\">" << std::endl;
+    *out << "    <div align=\"center\">" << std::endl;
+    *out << "        <form id=\"cal_select\">"<<std::endl;
+    *out << "            <div class=\"form-group row\">" << std::endl;
+    *out << "                <h2><span class=\"label label-info col-md-3\" id=\"cal_scurve\">Select calibration type:" << "</span></h2>" << std::endl;
+    *out << "                <div class=\"col-md-3\">" << std::endl;
+    *out << "                    <select class=\"form-control form-control-lg\" id=\"cal_type_select\" name=\"cal_type_select\" onChange=\"selectCalType()\">" << std::endl;
+    *out << "                        <option disabled selected value> -- select an option -- </option>" << std::endl;
+    *out << "                        <option>GBT Phase Scan</option>" << std::endl;
+    *out << "                        <option>Latency Scan</option>" << std::endl;
+    *out << "                        <option>S-curve Scan</option>" << std::endl;
+    *out << "                        <option>S-bit ARM DAC Scan</option>" << std::endl;
+    *out << "                        <option>ARM DAC Scan</option>" << std::endl;
+    *out << "                        <option>Derive DAC Trim Registers</option>" << std::endl;
+    *out << "                        <option>DAC Scan on VFAT3</option>" << std::endl;
+    *out << "                        <option>Calibrate CFG_THR_ARM_DAC</option>" << std::endl;
+    *out << "                        <option disabled value>Whatever else...</option>" << std::endl;
+    *out << "                    </select>" << std::endl;
+    *out << "                </div>" << std::endl; //<div class=\"col-md-6\">
+    *out << "            </div>" << std::endl; //<div class=\"form-group row\">
+    *out << "        </form>"<< std::endl;
+    *out << "    </div>" << std::endl;
+    *out << "</div>" << std::endl;
+
+    *out << "<div class=\"container col-md-12\" id=\"cal_interface\">" << std::endl;
+    *out << "</div>" << std::endl;
+    
+  
+} 
+
+void gem::calib::CalibrationWeb::applicationPage(xgi::Input* in, xgi::Output* out)
+{
+    CMSGEMOS_DEBUG("CalibrationWeb::applicationPage : Do nothing for the moment, will be eventually filled later");
+} ///CG : remove?
```

Comment:
The comments like this should use one of the standard tags and does not contain developer identifier:

- `NOTE`: Description of how the code works (when it isn't self evident).
- `XXX`: Warning about possible pitfalls, can be used as NOTE:XXX:.
- `HACK`: Not very well written or malformed code to circumvent a problem/bug. Should be used as `HACK:FIXME`:.
- `FIXME`: This works, sort of, but it could be done better. (usually code written in a hurry that needs rewriting).
- `BUG`: There is a problem here.
- `TODO`: No problem, but additional code needs to be written, usually when you are skipping something.